test/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ set(SRC
     conversions.cpp
     graph_rewrites/assign_clusters.cc
     graph_rewrites/deadness_test.cc
+    graph_rewrites/backend_manager_test.cc
```

Comment:
Minor: maybe move line `graph_rewrites/backend_manager_test.cc` before  `graph_rewrites/deadness_test.cc` just to keep it alphabetical like others?